tox.ini

```diff
@@ -12,7 +12,7 @@ deps =
 commands = nosetests {posargs}
 
 [testenv:flake8]
-deps = flake8==3.7.9
+deps = flake8==6.0.0
```

Comment:
I don't know exactly what these do, so I just set them to the same version as in the requirments file.